### ISSUE TITLE
Rec. 18: Deposit in Trusted Digital Repositories

### ISSUE THREAD

**Issue by sjDCC** [FAIR-Data-EG/Action-Plan#18]
Research data should be made available by means of Trusted Digital Repositories, and where possible in those with a mission and expertise to support a specific discipline or interdisciplinary research community.

- Policy should require data deposit in certified repositories and specify support mechanisms (e.g. incentives, funding of deposit fees, and training) to enable compliance.
**Stakeholders:** Policymakers; Funders; Publishers.

- Mechanisms need to be established to support research communities to determine the optimal data repositories and services for a given discipline or data type.
**Stakeholders:** Data services; Institutions; Data stewards.

- Concrete steps need to be taken to ensure the development of domain repositories and data services for interdisciplinary research communities so the needs of all researchers are covered.
**Stakeholders:** Data services; Funders; Institutions.

- Advocacy via scholarly societies, scientific unions and domain conferences is required so researchers in each field are aware of the relevant disciplinary repositories.
**Stakeholders:** Data services.

**Comment by jkh1** [comment]
I would use 'must' instead of 'should' or 'need'. Weak language and ambiguities create too many loopholes allowing stakeholders to wriggle out of their responsibilities.

**Comment by band** [comment]
Yes. Adopting the definitions in IETF RFC2119 (https://www.ietf.org/rfc/rfc2119.txt) can help with this problem.

**Comment by katerbow** [comment]
[DFG](http://www.dfg.de) position: In principle, depositing data in a certified repository would be ideal and it has to be welcomed. As for the moment, not many data repositories bear a quality certification or a seal of trust. It seems obvious to develop and to implement a widely accepted certification scheme before integrating requirements as depositing data in certified repositories in policies.

**Comment by Falco-KUB** [comment]
I'd add something about that recommended repositories should serve community needs and not commercial interests and avoid unfair publishers taking over the business (also in relation to Rec. 33).

**Comment by Eefkesmit** [comment]
Contribution on behalf of the **International Association of STM Publishers (STM)**:
As publishers we wish to promote and enable the use of trusted data repositories for datasets supporting publications, in conjunction with submission of manuscripts where appropriate – via recommended repository lists, services to help deposit data alongside the submission of manuscripts, and technological integrations between scholarly infrastructure, eg by means of API-standards.

STM and STM publishers offer to collaborate with repositories to achieve this.

**Comment by Drosophilic** [comment]
This work could draw on the relationships found within FAIRsharing.org, linking journal and funder data policies with data repositories and the standards they use.

**Comment by ScienceEurope** [comment]
Science Europe is working on a comprehensive list of criteria that trustworthy repositories should fullfill. This list of minimum criteria, which will be published towards the end of 2018, has been established after comparison of existing requirements. Some scientific communities use their discipline specific repositories or have already chosen a certified repository to work with, depending on their discipline-specific needs. Science Europe therefore does not recommend to refer to certain repositories nor a specific certification body, but instead provides a list of criteria to identify trustworthy repositories. Science Europe is also in contact with some well-known certification bodies to exchange on the draft criteria.

**Comment by ferag** [comment]
I think discipline dedicated trusted repositories are interesting but they need to assure their interoperability. However, I think some of the current protocols for interoperability (like OAI-PMH) are not enough to support complex interoperability actions.

**Comment by RCN2018** [comment]
It is important to bear in mind that the repositories are anchored in user-demands, and that there are not too many repositories, nationally nor internationally. We must avoid that services are duplicated. Some kinds of user fee or membership-fee could ensure that the users only wish to pay for the repositories that they find most useful (e.g. in combination with EOSC vouchers)

**Comment by jkh1** [comment]
A good repository is simply one whose data is actually used and key to a good quality repository is data curation. Therefore I don't see the need for any certification mechanism. 
I don't think a model based on access fees is going to work because it won't be sustainable. The preferred model should be to consider repositories as infrastructures.

**Comment by pkdoorn** [comment]
This recommendation mentions “trusted” and “certified” repositories, but avoids mentioning the CTS, which is in Rec. #10: **Trusted Digital Repositories** #10. Currently there are perhaps not yet enough CTS certified repositories, but we should work towards that.

**Comment by mromanie** [comment]
[ESO](www.eso.org) position
Please see our concerns on costs and funding in [Rec 10](https://github.com/FAIR-Data-EG/Action-Plan/issues/10).

**Comment by MSoareses** [comment]
On item 4 of this recommendation and in line with @Eefkesmit ‘s comment above publishers are stakeholders as their outreach to societies, scientific unions and at conferences. At Elsevier both through formal and informal ties to societies (via journals, conferences) and at discipline-specific conferences we are in a position to increase societies/unions awareness CoreTrustSea-certified repositories relevant to them.

**Comment by aidaturrini** [comment]
Comments above lead to #10 Rec. 10: Trusted Digital Repositories

**Comment by gtoneill** [comment]
Some overlap with Recommendation 10 related to trusted digital repositories. Perhaps merge?
